src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationlocation/InputAnnotationLocationPackageAnnotations.java

```diff
@@ -0,0 +1,17 @@
+package com.puppycrawl.tools.checkstyle.checks.annotation.annotationlocation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+class InputAnnotationLocationPackageAnnotations {
```

Comment:
please make a comment that this file is not an input, it is just place to define annotations to be used in package-info.java. Eventually we end up when config will be defined in each Input file,  but looks like we have exceptions from this rule for files like this .

Reply:
@romani @pbludov Secondary inputs can be placed in a special folder that will be skipped from validation and allowed to be named anything you want.
See https://github.com/checkstyle/checkstyle/blob/a73ff0890c27cbe8affc1a55f2710a231687bd85/src/test/java/com/puppycrawl/tools/checkstyle/internal/AllTestsTest.java#L258-L259 .

@romani Is this acceptable and you want a specific folder name? Other areas have used `inputs`.

Reply:
@pbludov , please try to move to subfolder "inputs"
example  - https://github.com/checkstyle/checkstyle/blob/master/src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/visibilitymodifier/inputs/InetSocketAddress.java

Reply:
done